sq/src/sq_cli.rs

```diff
@@ -637,6 +637,18 @@ $ sq key generate --userid \"<juliet@example.org>\" --userid \"Juliet Capulet\"
                                  "rev-cert",
                              ])
                              .requires("userid"))
+                        .arg(Arg::with_name("primary-flags")
```

Comment:
I'm not sure if this is the best name for this, since you're also implicitly controlling what subkeys to use here

Reply:
The flag is to control only the type of primary key we will be creating. As mentioned above, we are not exposing support to create multiple signing and/or encryption subkeys (for now).

Reply:
I think it is better to use `--key-flags` instead, for later flexibility.
E.g.
```
--key-flags=CS,EtEr => vec![cert&sign, et&er] (two-key)
--key-flags=C,S,EtEr => vec![cert, sign, et&er] (three-key)
--key-flags=anything-else => error, not supported today
```

Reply:
Renamed `primary-flags` to `key-flags`

Reply:
I think this has been handled properly now; resolving